controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/noderepository/NodeRepositoryNode.java

```diff
@@ -347,6 +351,22 @@ public void setMinCpuCores(Double minCpuCores) {
         this.minCpuCores = minCpuCores;
     }
 
+    public Double getBandwidth() {
+        return bandwidth;
+    }
+
+    public void setBandwidth(Double bandwidth) {
```

Comment:
There should be some indication of what units this is in. Gbps, Mbps?

Reply:
Yeah, right now it just follows the name if has in the node-repository. We should just rename that entire field.